packages/bng-metric-service/src/helpers/extractors/extraction-config/dirname.cjs

```diff
@@ -0,0 +1 @@
+module.exports = __dirname
```

Comment:
@paulsimonandrews  - This file is new. As multiple modules contain a file like this there is some technical debt to refactor this module into the utilities library for use by ES modules.